boards/arm/ubx_bmd345eval_nrf52840/Kconfig

```diff
@@ -0,0 +1,8 @@
+# BMD-345-EVAL board configuration
+
+# Copyright (c) 2021 u-blox AG
+# Copyright (c) 2021 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+# No special configuration is required.
```

Comment:
nrf52840 chip itself have on board DC to DC regulator. Its need to be enabled to handle low power modes.

Reply:
The nRF52840 has a DC-DC converter, but it requires external inductors in order to use it. The BMD-345 module does not have these inductors and no means to add them due to the space required on the module for the RFX2411 FEM. Enabling the DC-DC converter will cause an application to fail. Only the default LDO within the nRF52840 may be used.

Reply:
if I understand correctly, there is no LC circuit between the DCCH, VDD and DCC, DEC4. That's why we don't need to enable the DC/DC regulator.

Reply:
Correct